vllm/worker/worker.py

```diff
@@ -180,6 +184,24 @@ def _prepare_inputs(
             prompt_len = len(prompt_tokens)
             prompt_lens.append(prompt_len)
 
+            assert len(seq_ids) == 1, "Prompt input should have only one seq."
+            if sampling_params.prompt_logprobs is not None:
+                selected_token_indices.extend(
+                    range(selected_token_start_idx,
+                          selected_token_start_idx + prompt_len - 1))
+            selected_token_indices.append(selected_token_start_idx +
+                                          prompt_len - 1)
+            selected_token_start_idx += prompt_len
```

Comment:
To adapt to the changes in #1381:
```suggestion
            selected_token_start_idx += input_metadata.max_prompt_len
```